options/posix/include/netdb.h

```diff
@@ -105,6 +105,7 @@ int getaddrinfo(const char *__restrict, const char *__restrict,
 		const struct addrinfo *__restrict, struct addrinfo **__restrict);
 struct hostent *gethostent(void);
 struct hostent *gethostbyname(const char *);
+struct hostent *gethostbyname2(const char *, int);
```

Comment:
where's the implementation?

Reply:
In `options/posix/generic/netdb-stubs.cpp`.

Reply:
oh, huh, I didn't see it